packages/daemon/src/runner/helper.ts

```diff
@@ -13,8 +14,13 @@ import {
 
 const { PLUGINS, PIPCOOK_LOGS } = constants;
 
-export async function parseConfig(configPath: string, generateId = true): Promise<PipelineDB> {
-  const configJson: RunConfigI = await fs.readJson(configPath);
+export async function parseConfig(config: string, generateId = true): Promise<PipelineDB> {
+  let configJson: RunConfigI = null;
+  if (/^https?:\/\/.+/.test(config)) {
+    configJson = JSON.parse(await request(config));
```

Comment:
Maybe we need a `loadJson()` at utils?

Reply:
Use `url.parse()` to get the url protocol.